src/picongpu/include/plugins/IsaacPlugin.hpp

```diff
@@ -400,12 +403,10 @@ class IsaacPlugin : public ILightweightPlugin
                 json_object_set_new( visualization->getJsonMetaRoot(), "cell count", json_string( "Total numbers of cells" ) );
                 json_object_set_new( visualization->getJsonMetaRoot(), "particle count", json_string( "Total numbers of particles" ) );
             }
-            if (visualization->init() != 0)
+            if (visualization->init(reconnect?RetryEverySend:ReturnAtError) != 0)
```

Comment:
I wondered, whether you will notice, that my template parameter is no constant at all. ^^